client/client.py

```diff
@@ -0,0 +1,24 @@
+import socket
+
+class Connection:
+  sock = None
```

Comment:
These are static members, you don't need to specify them. Just assign inside init